src/keyword_vpd_parser.cpp

```diff
@@ -39,7 +39,7 @@ types::VPDMapVariant KeywordVpdParser::parse()
                 throw(DataException("Invalid Keyword Vpd Start Tag"));
             }
         }
-        auto kwValMap = populateVpdMap();
+        types::KeywordVpdMap kwValMap = populateVpdMap();
```

Comment:
Do we still need this change? If not then get rid of unrelated changes.

Reply:
done